### ISSUE TITLE
Post processing rarely works

### ISSUE THREAD

**Issue by mlg9000** [DobyTang/LazyLibrarian#489]
It seems that the post processing in Lazy Librarian rarely works.  Of the ~50 books/magazines I've tested it was only able to determine 2.  Many/most of the books had the full author and book names very cleanly/correctly listed in the filenames directly, magazines the name and date, and the score that shows up in the debug logs is still very low (~25-50% match range).  I've tested renaming downloads with various iterations of very specific named files and it makes no difference.  It effectively doesn't work from what I can tell.

**Comment by philborman** [comment]
The name matching code is quite simple and uses a fuzzy matching library.
It takes each word in the source and target and works out the percentage
match,  then deducts a point for each extra or missing word.  Some specific
examples of failures with debug log might help us improve.  I typically get
matches in the over 95% range so it would be interesting to see why your
matches are so low.

**Comment by philborman** [comment]
Changed the way torrent name matching works, hopefully will improve your issue. Depends hugely on the person uploading the torrent being sensible about naming.
Usenet is much simpler, the downloaded filename always matches what you asked for. I tend to use usenet, with torrents as a fallback, so hadn't hit this issue until I went looking for it.
Torrents have a title, which is what we search, and a filename that might or might not be the same, eg if you search limetorrents for "Henri Charriere Papillon" you get a title of "Henri Charriere - Papillon", 100% match, but it will send you a file called HC_pap.zip which you and I might guess is the right file, but lazylibrarian can't. Or at least couldn't until this release. If you still have postprocessing failures, a debug log of the postprocessing would help.
